.travis.yml

```diff
@@ -21,6 +21,9 @@ env:
   - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
 
 install:
+  #  FIXME: remove this after merge
+  - git clone --branch 12.0-recurrence-mechanism-on-contract-line https://github.com/sbejaoui/contract.git PR207
```

Comment:
Why this depends on that PR?

Reply:
cc @mreficent this seems the method for including other PRs of the same branch.

Reply:
@pedrobaeza To benefit from the refactoring of `contract.py` (`_finalize_invoice_creation`). This hack in `.travis.yml` must be reverted after merge.

Reply:
I would want to merge this module migration, but without having to depend on the other one. Is it possible?

Reply:
We should finalize and merge https://github.com/OCA/contract/pull/207 and then merge this one since my work here is based on the refactoring (which include breaking changes).

Reply:
OK, as you know, that one is very huge, so I wanted to speed up this migration. Putting again wip label here.

Reply:
Yes it is huge. But, IMO it is top priority because several PR's depend on it.